app/Http/Controllers/api/ModalityController.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Http\Controllers\api;
+
+use App\Http\Controllers\Controller;
+use App\Models\Resume;
+
+
+class ModalityController extends Controller
+{
+    public function __invoke($studentId)
+    {
+        try {
+            $resume = Resume::where('student_id', $studentId)->first();
```

Comment:
Extract this logic from the controller. A controler should be only using a service and managing the output of application

Reply:
Ok